jobs/service-fabrik-broker/spec

```diff
@@ -232,6 +249,8 @@ properties:
     default: 3
   backup.provider:
     description: "IaaS-specific backup provider configuration"
+  ha.provider:
```

Comment:
Why is it part of broker spec and not keepalived spec? I see it is declared here but used in the keepalived job